staging/publishing/rules.yaml

```diff
@@ -397,8 +397,6 @@ rules:
       branch: master
     - repository: kms
       branch: master
-    - repository: code-generator
-      branch: master
```

Comment:
code-generator needs to be listed as a dependency due to https://github.com/kubernetes/kubernetes/commit/321095dccb60a14e4a886296014ff014759f52b9

Reply:
I think this PR drops the code-generator-using scripts from those repos, so they no longer have content that references code-generator, so this looks right

Reply:
correct - I intend to retool this all in go workspaces conversion, to erase any dependencies on GOPATH or `../../../../` sorts of trickery